### DISCUSSION TITLE
我平时的Obsidian写在mac的机器上, 打开很久没有用的windows电脑想要同步一下, 发现旧数据把新数据给覆盖了

### DISCUSSION THREAD

**Discussion by jensenojs** [remotely-save/remotely-save#734] [Q&A]
如题, 有回滚的机制嘛?

**Comment by fyears** [comment]
obsidian自己有历史版本，可以看看能否救一下。

此外 remotely save 也有修改百分比保护和智能合并功能，可以开启来避免下一次类似问题

**Comment by jensenojs** [comment]
问题解决了!

***

答主说的是history plan.

- https://help.obsidian.md/Obsidian+Sync/Version+history

***

我的老电脑上面的插件没有更新, 修改百分比的机制和本插件升级之后要求数据备份的warming没有及时看到, 是我的锅. 

我当时迷糊地把note直接放在OneDrive里面了(现在想想放在了OneDrive之后为什么还要用这个插件有点不记得了, 可能是为了在ipad/iphone上也能看到)

最终我的fix是通过oneDrive还原早些时间点的状态的方式来做到的, 这样只丢失了一天的笔记. 注意这个功能的使用需要office 365, 我为此充了一个月的会员, 链接留在下面以供有需要的人参阅. 

https://support.microsoft.com/zh-cn/office/还原-onedrive-fa231298-759d-41cf-bcd0-25ac53eb8a15

@fyears  非常感谢您的支持
